hw6/services/authentication.service.ts

```diff
@@ -0,0 +1,17 @@
+import { sign, verify } from 'jsonwebtoken';
+
+export class AuthenticationService {
+    static getToken(payload: { name: string | undefined, id: string | undefined }): string {
+        return sign(payload, process.env.JWT_SECRET || 'secret', { expiresIn: '1h' });
+    }
+
+    static checkToken(token: string): boolean {
+        try {
+            const decoded = verify(token, process.env.JWT_SECRET || 'secret');
```

Comment:
Better not to default such values as secrets. It would prevent anyone to misconfigure the app in real.